core/src/main/java/org/elasticsearch/action/DocWriteResponse.java

```diff
@@ -242,7 +242,7 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
         if (forcedRefresh) {
             builder.field("forced_refresh", forcedRefresh);
         }
-        shardInfo.toXContent(builder, params);
+        builder.field("_shards", shardInfo);
```

Comment:
ah right because otherwise it is not a valid json alone. I don't know, I think that if we touch all these little things as part of this parsing effort it becomes risky. I did that myself in my PR and I am thinking of reverting it. Maybe we should just accept that toXContent may print something that's not a valid json alone, that's it. It kinda makes sense that shard info starts its own object with its own name. I find it odd when toXContent simply starts fields without an ancestor.  What do you think?